connectors/idp-extraction/src/main/java/io/camunda/connector/idp/extraction/model/ExtractionResult.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.idp.extraction.model;
+
+public record ExtractionResult(String response) {}
```

Comment:
Using an `Object` would allow to return a json object.

Reply:
Done.